.yarnrc

```diff
@@ -1,3 +1,3 @@
 # See https://yarnpkg.com/en/docs/yarnrc#yarn-path-
 # TL;DR; We want to ensure that everyone uses the same yarn version, both locally and on CI.
-yarn-path ".yarn/releases/yarn-1.16.0.js"
+yarn-path ".yarn/releases/yarn-1.17.2.js"
```

Comment:
@emmenko updated to the recent version here. 1.16.0 is already a bit old.

Reply:
Awesome!